### PR TITLE
net/siproxd: fixed model and save buttons

### PR DESCRIPTION
**Pull request by mimugmail**
@fichtner can you help me with the save buttons?

1. It's not in the middle, I'm not sure why:
image

2. It's working with the save buttons, but I left formid as general, is this ok?

### CODE REVIEW

File: net/siproxd/src/opnsense/mvc/app/views/OPNsense/Siproxd/general.volt [hunk 1]
```diff
@@ -86,6 +90,10 @@ POSSIBILITY OF SUCH DAMAGE.
                 </tr>
             </tfoot>
         </table>
+        <hr />
```
Comment (fichtner) [inline net/siproxd/src/opnsense/mvc/app/views/OPNsense/Siproxd/general.volt]: &lt;hr /&gt; belongs into the div below, alignment is usually fixed with &lt;br /&gt;&lt;br /&gt; after the button in other forms

File: net/siproxd/src/opnsense/mvc/app/views/OPNsense/Siproxd/general.volt [hunk 2]
```diff
@@ -64,6 +64,10 @@ POSSIBILITY OF SUCH DAMAGE.
                 </tr>
             </tfoot>
         </table>
+        <div class="col-md-12">
+            <button class="btn btn-primary"  id="saveAct_user" type="button"><b>{{ lang._('Save') }}</b><i id="saveAct_user_progress" class=""></i></button>
+        <hr />
```
Comment (fichtner) [inline net/siproxd/src/opnsense/mvc/app/views/OPNsense/Siproxd/general.volt]: hr before button br br after ;)

**Comment by mimugmail** [comment]
able to merge now :)

**Comment by fichtner** [comment]
As always: thanks!

**Pull request merged**